apps-rendering/src/components/headline.tsx

```diff
@@ -183,33 +185,53 @@ const headlineStyles = (format: ArticleFormat): SerializedStyles => {
 		background-color: ${background.headline(format)};
 		position: relative;
 		white-space: pre-wrap;
-		padding: 0 ${remSpace[1]};
+		padding: 0 ${remSpace[3]} ${remSpace[1]};
 		display: inline;
 		box-decoration-break: clone;
+
+		${from.wide} {
+			padding: 0 ${remSpace[2]} ${remSpace[1]};
+		}
+
+		${darkModeCss`
+			background-color: ${background.headlineDark(format)};
+		`};
 	`;
 };
 
+const interviewHeadlineStyles: SerializedStyles = css`
+	padding-left: 0;
+	padding-right: ${remSpace[12]};
+	margin-top: -2.14rem;
+	position: relative;
+
+	${from.wide} {
+		margin: -2.14rem auto 0;
+	}
```

Comment:
Can I suggest an alternative to negative margins? They can be a little brittle (particularly with a very specific value like `2.14rem`) and can be difficult to debug for others looking at this CSS down the line.

What you could do instead is add the following to the `HeadlineTag`:

```css
position: absolute;
transform: translateY(-100%);
```

and this to `HeaderImage` when `Design.Interview`:

```css
margin-bottom: 0;
```

Here's a useful piece on CSS positioning: https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Positioning

Here's are the docs on `translateY` - the nice thing here is that, unlike a lot of percentages in CSS, the percentage refers to the _height_ rather than the _width_, so it's possible to translate upwards by exactly the height of the element, regardless of what that value is and whether it changes: https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translateY

Reply:
Also cc @mxdvl because he's knowledgeable about this sort of thing and may have a better suggestion than me 🙂

Reply:
Good shout, thanks!